src/main/webapp/style.css

```diff
@@ -13,17 +13,54 @@ select:focus {
   outline: none;
 }
 
-/* Horizontally and vertically center */
 .wrapper {
-  left: 50%;
-  position: absolute;
-  top: 50%;
-  transform: translate(-50%, -50%);
+  margin: 0 60px 0 60px;
 }
 
-#map {
-  position: relative;
-  width: 626px;
-  height: 391px;
-  font-size: 13px;
+#header {
+  border-bottom: 1px solid #e7e7e9;
+  line-height: 1em;
+  padding-left: 30px;
+}
+
+#query {
+  text-align: center;
+}
+
+input[type=submit] {
+  background: #bfbfbf;
```

Comment:
This makes the button seem grayed out which makes seem like it is not clickable.

Reply:
Changed button color to green.